core/validation/failures.py

```diff
@@ -511,15 +511,17 @@ def failures_to_messages(
     # one pre-transformation failure means payload is entirely pre-transformation failures
     if any(isinstance(failure, PreTransFormationFailure) for failure in validation_failures):
         # ignore tab and section for pre-transformation failures
+        # tab, section, message
         return {"PreTransformationErrors": [message for _, _, message in error_messages]}
 
     # remove duplicate failure messages
     error_messages = list(set(error_messages))
     error_messages.sort()
 
-    # group by tab and section
-    failures_grouped_by_tab = group_by_first_element(error_messages)
-    failures_grouped_by_tab_and_section = {
-        tab: group_by_first_element(failures) for tab, failures in failures_grouped_by_tab.items()
-    }
-    return {"TabErrors": failures_grouped_by_tab_and_section}
+    validation_errors = []
+    for tab, section, message in error_messages:
+        validation_errors.append(
+            {"tab": tab, "section": section, "cell": "A1", "description": message}  # hardcode this for now
```

Comment:
Should the cell not be a variable? Are we going to do this in a subsequent PR? If so can we make the comment a TODO?